articles/multifactor-authentication/administrator/customizing-widget.md

```diff
@@ -0,0 +1,13 @@
+---
+description: How to Customize the Guardian Widget
+---
+## Customizing the Guardian Screen
+
+You may change the logo and the friendly name that is displayed to your users. To do so, make the appropriate setting changes from the Guardian page's link to [Account Settings](${manage_url}/#/account). You can also reach the Account Settings page by clicking on your user name on the top right of the page and then selecting **Account Settings** from the dropdown menu.
+
+![](/media/articles/mfa/guardian-logo-and-name-settings.png)
+
+ * **Friendly Name**: the name of the app that you want displayed to users
+ * **Logo URL**: the URL that points to the logo image you want displayed to users
+
+Auth0 recommends using a logo image that is at least 100x100 pixels, although an image that is 200x200 pixels ensures quality viewing in devices with Retina or high DPI displays.
```

Comment:
Keep in mind that the same logo is being used for the error page, not sure if you took that into account for the advised sizes